crates/re_ui/src/command.rs

```diff
@@ -181,6 +185,17 @@ impl UICommand {
                 "Print datastore",
                 "Prints the entire data store to the console. WARNING: this may be A LOT of text.",
             ),
+            #[cfg(not(target_arch = "wasm32"))]
+            Self::ClearPrimaryCache => (
+                "Clear primary cache",
+                "Clears the primary cache in its entirety.",
+            ),
+            #[cfg(not(target_arch = "wasm32"))]
+            Self::PrintPrimaryCache => (
+                "Print primary cache",
+                "Prints the state of the entire primary cache to the console. WARNING: this may be A LOT of text.",
```

Comment:
You could consider having the command copy it to the clipboard instead (or also)